deny.toml

```diff
@@ -53,4 +50,12 @@ ignore = [
     # alert is received during a handshake, `complete_io` does not terminate.
     # Vulnerable version only used in dev-dependencies
     "RUSTSEC-2024-0336",
+
+    # unmaintained dependencies
```

Comment:
Unmaintained dependencies are now reported as an error. There doesn't appear to be a way to disable this.